packages/vocabularies-types/src/Edm.ts

```diff
@@ -564,6 +568,7 @@ export type RawSchema = {
 
 export type RawAction = RemoveAnnotationAndType<Action>;
 export type RawActionImport = RemoveAnnotationAndType<ActionImport>;
+export type RawActionParameter = RemoveAnnotationAndType<ActionParameter>;
```

Comment:
This type is not used by other types in here. Wouldn't it make sense to use it as the type of `RawAction["parameters"]` somehow?

Reply:
yes make sense :)